src/astarte-client/models/Mapping/index.ts

```diff
@@ -159,6 +140,17 @@ export class AstarteMapping {
     this.documentation = validatedObj.documentation;
   }
 
+  static isEquivalentEndpoint(endpoint1: string, endpoint2: string): boolean {
+    const endpoint1Parts = endpoint1.split('/');
+    const endpoint2Parts = endpoint2.split('/');
+    if (endpoint1Parts.length !== endpoint2Parts.length) {
+      return false;
+    }
+    return endpoint1Parts.every((part, index) =>
```

Comment:
That's the reason why I first inlined the helper functions, before your comment about extracting them out to define them once. However I would keep the helper functions defined since it aids with readability

Reply:
i mean, you could just
```
const matchEndpointPart = ([ part1, part2]) => {
}

[...]

_.zip(endpoint1.split('/'), endpoint2.split('/')).every(matchEndpointPart)
```
no lambas involved

Reply:
Feels odd how `matchEndpoint` accepts two params and `matchEndpointPart` accepts an array param.
However we are not exposing `matchEndpointPart` and I like how terse and readable it becomes, you scored a point here.